From 0493b1107b9cc5f24786c15e250f32dd82ad9f6e Mon Sep 17 00:00:00 2001
From: Venkateshwar Rao G <venkateshwar.rao.gannavarapu@xilinx.com>
Date: Wed, 14 Mar 2018 21:02:33 +0530
Subject: [PATCH 0333/1851] dt: bindings: display: xlnx: Adding gpio reset
 entry

commit 5be4f9ad16c976cff6ca9fd38c141eb5f9385cfc from
https://github.com/Xilinx/linux-xlnx.git

This patch adds the gpio reset entry for vpss color space converter.
Moved to bindings/display/xlnx folder.

Signed-off-by: Venkateshwar Rao G <venkateshwar.rao.gannavarapu@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../bindings/{drm/xilinx => display/xlnx}/xlnx,vpss-csc.txt    | 3 +++
 1 file changed, 3 insertions(+)
 rename Documentation/devicetree/bindings/{drm/xilinx => display/xlnx}/xlnx,vpss-csc.txt (85%)

diff --git a/Documentation/devicetree/bindings/drm/xilinx/xlnx,vpss-csc.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-csc.txt
similarity index 85%
rename from Documentation/devicetree/bindings/drm/xilinx/xlnx,vpss-csc.txt
rename to Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-csc.txt
index 7b1e29870b96..7991e3e96937 100644
--- a/Documentation/devicetree/bindings/drm/xilinx/xlnx,vpss-csc.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-csc.txt
@@ -13,9 +13,12 @@ Required properties:
   width expressed as a number of bits per pixel component. Supported video
   width values are 8/10/12/16.
 
+-reset-gpios: GPIO specifier to assert/de-assert the reset line.
+
 Example:
 	csc@a0040000 {
 		compatible = "xlnx,vpss-csc";
 		reg = <0x0 0xa0040000 0x0 0x10000>;
+		reset-gpios = <&gpio 0x0 GPIO_ACTIVE_LOW>;
 		xlnx,video-width = <8>;
 	}
-- 
2.31.1

